src/System.Management.Automation/engine/Utils.cs

```diff
@@ -538,8 +556,148 @@ internal static bool IsValidPSEditionValue(string editionValue)
                     if (result != null) { return (T) result; }
                 }
             }
+
+            return null;
+        }
+
+#if !UNIX
+        private static readonly Dictionary<string, string> GroupPolicyKeys = new Dictionary<string, string>
+        {
+            {nameof(ScriptExecution), @"Software\Policies\Microsoft\PowerShellCore"},
+            {nameof(ScriptBlockLogging), @"Software\Policies\Microsoft\PowerShellCore\ScriptBlockLogging"},
+            {nameof(ModuleLogging), @"Software\Policies\Microsoft\PowerShellCore\ModuleLogging"},
+            {nameof(ProtectedEventLogging), @"Software\Policies\Microsoft\Windows\EventLog\ProtectedEventLogging"},
+            {nameof(Transcription), @"Software\Policies\Microsoft\PowerShellCore\Transcription"},
+            {nameof(UpdatableHelp), @"Software\Policies\Microsoft\PowerShellCore\UpdatableHelp"},
+            {nameof(ConsoleSessionConfiguration), @"Software\Policies\Microsoft\PowerShellCore\ConsoleSessionConfiguration"}
+        };
+        private readonly static ConcurrentDictionary<Tuple<ConfigScope, string>, PolicyBase> s_cachedPoliciesFromRegistry =
```

Comment:
Really, we should register for GP change notifications and clear the cache.   Not a regression, so, can you add a comment or file an issue:  https://msdn.microsoft.com/en-us/library/windows/desktop/aa374404(v=vs.85).aspx

Reply:
Issue https://github.com/PowerShell/PowerShell/issues/5804 is opened to track this.